.github/docker-soak-tests/load-generator/Dockerfile

```
@@ -0,0 +1,11 @@
+FROM golang:1.16
+
+WORKDIR $GOPATH/main
+
+ARG GOPROXY=direct
+
+ENV GOPROXY=direct
+
+RUN go get github.com/rakyll/hey
```

Download one of the precompiled (`COPY` command can reference a URL too I think)

https://github.com/rakyll/hey#installation

---

Sounds good, I was able to downloaded it directly by changing the line to be like so:

```suggestion
RUN curl https://hey-release.s3.us-east-2.amazonaws.com/hey_linux_amd64 -o /usr/local/bin/hey && chmod +x /usr/local/bin/hey
```